superset-frontend/src/cccs-viz/plugins/plugin-chart-ag-grid/src/types/DomainValueRenderer.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+
+export default class DomainValueRenderer extends Component<
+  {},
+  { cellValue: any }
+> {
+  constructor(props: any) {
+    super(props);
+
+    this.state = {
+      cellValue: DomainValueRenderer.getValueToDisplay(props),
+    };
+  }
+
+  // update cellValue when the cell's props are updated
+  static getDerivedStateFromProps(nextProps: any) {
+    return {
+      cellValue: DomainValueRenderer.getValueToDisplay(nextProps),
+    };
+  }
+
+  render() {
+    return (
+      <span>
+        <a href="http://10.162.232.22:8000/gwwk.html">{this.state.cellValue}</a>
```

Comment:
I'm sure this has been there for a while but this is bad.

Reply:
I am not sure what this was supposed to link to but I am going to remove it